derive/src/attributes/mod.rs

```diff
@@ -0,0 +1,2 @@
+mod row;
+pub use row::Row;
```

Comment:
`pub(crate)` (and all below). It seems this subcrate doesn't use https://github.com/ClickHouse/clickhouse-rs/blob/main/Cargo.toml#L17